### ISSUE TITLE
roots.cx

### ISSUE THREAD

**Issue by janwirth** [leonidas/gulp-project-template#7]
https://github.com/jenius/roots
basically, for me, this is just what i have been looking for.
neat environment.

**Comment by FranzSkuffka** [comment]
no comment?

**Comment by rikukissa** [comment]
Good that you've found something that fits your needs. [Yeoman](http://yeoman.io/) also providers a large number of project templates. https://github.com/leonidas/gulp-project-template/issues/13#issuecomment-66960375

**Comment by FranzSkuffka** [comment]
To me, it doesn't seem like you took a look at it, especially regarding the topic of that issue discussion. It features the full jade/cs/styl stack whilst maintaining backwards compatibility with html, css and js in addition to excluding all the build logic from the project directory (gulpfile, nodemodules etc.).

Don't mean to be a dick but rather think that we have similar goals..

**Comment by rikukissa** [comment]
Yeah sure, I totally understand! I've looked into roots before and tried it too, but I just didn't see it as something worth switching to. In your opinion, what are the main advantages of using roots? How is it better than just using [webpack](http://webpack.github.io/) for example?

**Comment by FranzSkuffka** [comment]
i don't know webpack, to be honest, but to me it seems like a hugely complicated thing. With roots you get a simple to set up and use, fully functional with templating and preprocessing, yet easily extendable frontend full stack.
I was rather comparing it to the gulp-project template. I was looking for exactly this kind of fullstack a year ago and got here first, being overly happy i found exactly what i needed. A week later i found roots and realised that jenius has exactly the same idea of the perfect frontend stack as we do, however with the knowhow of his colleagues and company ressources he was able to wrap up and perfect it.

There are no needs left unsatisfied as there is a plugin for content collections, stylus libraries for common use, breakpoints and grid system are provided out of the box, just like an autoprefixer, requirejs for frontend etc. In addition to that it has routines for deployment built in, it is compatible with a wide variety of languages and the build system is omitted.

In short terms: your project is exactly what i was looking for, i was working on basically the same project before i found it, but no offense, roots just excels. Give it a try. It would be great to have you onboard.
